packages/providence/README.md

```diff
@@ -0,0 +1,215 @@
+# Providence
+
+Providence is the 'All Seeing Eye' that generates usage statistics by analyzing code.
+It measures the effectivity and popularity of your software.
+In just a few clicks you can measure the impact for (breaking) changes; making
+your release process more stable and predictable.
+
```

Comment:
I think we should add a small section about who the main user is, and an example analysis of what they can do.

Reply:
Good idea